coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -9526,6 +9526,54 @@ def forward(self, x):
             compute_unit=compute_unit
         )
 
+class TestISTFT(TorchBaseTest):
+    @pytest.mark.slow
+    @pytest.mark.parametrize(
+        "compute_unit, backend, input_shape, complex, n_fft, hop_length, win_length, window, center, pad_mode, normalized, onesided, length",
+        itertools.product(
+            compute_units,
+            backends,
+            [(1, 32, 9), (32, 9), (3, 32, 9)], # input shape
+            [False, True], # complex
+            [16], # n_fft
+            [None, 4, 5], # hop_length
+            [None, 16, 9], # win_length
+            [None, torch.hann_window], # window
+            [None, False, True], # center
+            ["constant", "reflect", "replicate"], # pad mode
+            [False, True], # normalized
+            [None, False, True], # onesided
+            [None, 60], # length
+        )
+    )
+    def test_istft(self, compute_unit, backend, input_shape, complex, n_fft, hop_length, win_length, window, center, pad_mode, normalized, onesided):
```

Comment:
This test is great! Besides this, as `istft` is the inverse of `stft`, I'm wondering if it's possible to add another test case, where the torch model first call `torch.stft`, and then feed the output back to `torch.istft` to restore the original input?